### PR TITLE
fix up some comments

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

File: deploy/kiali/kiali_cr.yaml [hunk 1]
```diff
@@ -22,16 +22,16 @@ spec:
...
-# It can typically be used to inject some CI meta-data or documentation links into Kiali views.
+# A list of additional details that Kiali will look for in annotations and display, for every workload and service, in their respective details pages.
+# It can typically be used to inject some CI metadata or documentation links into Kiali views.
 # Each item in the list is an object with "annotation", "title" and "icon_annotation" fields to indicate which annotation Kiali needs to look for, and how it should be displayed.
 # "icon_annotation" is optional and would display an icon next to the text.
-# At the moment, the value of the icon annotation can only be one of "rest", "grpc" or "graphql", or be ignored.
-# By default, this settings recognizes API Documention links via annotation "kiali.io/api-spec" and icon-annotation "kiali.io/api-type".
+# At the moment, the value of the icon annotation can only be one of "rest", "grpc" or "graphql"; otherwise, it is ignored.
+# By default, these settings recognize API Documention links via annotation "kiali.io/api-spec" and icon-annotation "kiali.io/api-type".
```
Comment (jshaughn) [inline deploy/kiali/kiali_cr.yaml:34]: Documention -> Documentation

File: deploy/kiali/kiali_cr.yaml [hunk 2]
```diff
@@ -96,7 +92,7 @@ spec:
 #
 # Allows for controlling what namespaces/projects are returned by Kiali.
 #
-# 'exclude' is optional takes an list of namespace/projects to be excluded from the list
+# 'exclude' is optional and takes a list of namespaces to be excluded from the list
```
Comment (jshaughn) [inline deploy/kiali/kiali_cr.yaml:95]: Does `exclude` support regex?

**Review by jshaughn — CHANGES_REQUESTED** [review]
small comments

File: deploy/kiali/kiali_cr.yaml (hunk 2 above)
Comment (jmazzitelli) [reply deploy/kiali/kiali_cr.yaml:95]: I think. I dunno.  All I did was change this to be proper English.

File: deploy/kiali/kiali_cr.yaml (hunk 1 above)
Comment (jmazzitelli) [reply deploy/kiali/kiali_cr.yaml]: done.

File: deploy/kiali/kiali_cr.yaml (hunk 2 above)
Comment (jmazzitelli) [reply deploy/kiali/kiali_cr.yaml:95]: Yes, it says so in the following sentence. And we know it does because the default value uses regex.

**Review by jshaughn — APPROVED** [review]
LOK

**Pull request merged**